extensions/measurement-tracking/src/panels/PanelMeasurementTableTracking/index.js

```diff
@@ -102,6 +105,35 @@ function PanelMeasurementTableTracking({ servicesManager, commandsManager }) {
 
   const activeMeasurementItem = 0;
 
+  const onExportClick = () => {
```

Comment:
`onExportClick` is the name of the prop. For it to read smoother you can name this `localDownloadStructuredReport`

So the prop binding reads:

`onExportClick={localDownloadStructredReport}`

Reply:
If this doesn't depend on reactive values, you don't need to define this function in the component. Re-renders will recreate it. You can shift it outside of the component in the same file, or a separate file.